kinto_attachment/utils.py

```diff
@@ -87,8 +87,10 @@ def patch_record(record, request):
     # Simulate update of fields.
     request.validated = record
     request.body = json.dumps(record).encode('utf-8')
+    request.bound_data['attachment_auto'] = True
```

Comment:
I just said that because I feel it could be possible to set `attachment_auto` in the payload to let a user update it nevertheless but could be wrong.